### PR TITLE
[Docs] fix links

### PR DESCRIPTION
**Pull request by ddhodge**
Fixed broken links

### CODE REVIEW

File: docs/content/latest/explore/security/security.md [hunk 1]
```diff
@@ -41,13 +41,13 @@ You can choose the method to use to authenticate a particular client connection
 
 YugabyteDB stores authentication credentials internally in the YB-Master system tables. The authentication mechanisms available to clients depend on what is supported and exposed by the YSQL, YCQL, and YEDIS APIs.
 
-Read more about [how to enable authentication in YugabyteDB](../../secure/authentication).
+Read more about [how to enable authentication in YugabyteDB](/latest/secure/authentication).
```
Comment (polarweasel) [inline docs/content/latest/explore/security/security.md:44]: Avoid absolute links whenever you can. They'll break later when this becomes a numbered version.

**Review by polarweasel — CHANGES_REQUESTED** [review]
No absolute links!

File: docs/content/latest/explore/security/security.md (hunk 1 above)
Comment (polarweasel) [reply docs/content/latest/explore/security/security.md:44]: (Each link just needs one more `../` in it and they should all work)

Comment (ddhodge) [reply docs/content/latest/explore/security/security.md:44]: fixed

**Pull request merged**